src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -370,7 +371,8 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper()
             .registerModule(new ParameterNamesModule())
             .registerModule(new Jdk8Module())
-            .registerModule(new JavaTimeModule());
+            .registerModule(new JavaTimeModule())
+            .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_VALUES, true);
```

Comment:
As I commented on the other PR, I don't think we want to change our global objectmapper strategy to support one  specific case. Why is this needed? Can we fix it by making the code specific to the upper/lower case instead? If not, can we isolate this change in behavior to the one or two cases where it's needed?

Reply:
Doing this allows us to not sprinkle `toUpper/Lowercase` throughout our code. For enums that are generated using OpenAPI, we need to lower the case because they create a method annotated with `@JsonCreator` that compares the value of the enum member instead of the name. We store the name in our DB, which works fine for enums without the OpenAPI-generated `@JsonCreator` method. This just feels cleaner as a way to harmonize the two cases without sprinkling methods throughout our code than might need changing or become obsolete if we migrate an enum in/out of OpenAPI.

Reply:
In the current code base I can only find a single case where `toLowerCase()` is used to work around this problem, in `OpenApiEnumConverter.convert()`. I don't see any cases of `toUpperCase()` that would be affected by this. So I don't see that doing this would have a large benefit for us.

I think changing this would also affect how all our openapi inputs are processed, and allow upper and lower case fields for all struct inputs. Is this something we want to change? @nmalfroy

Reply:
Yeah, I'm kinda tempted to leave this bit of the change out

Reply:
I've created a dao-specific object mapper to avoid these issues.